pkg/exposer/csi_snapshot.go

```diff
@@ -174,12 +174,14 @@ func (e *csiSnapshotExposer) Expose(ctx context.Context, ownerObject corev1.Obje
 	// for backupPVC (intermediate PVC in snapshot data movement) object creation
 	backupPVCStorageClass := csiExposeParam.StorageClass
 	backupPVCReadOnly := false
+	spcNoRelabeling := false
 	if value, exists := csiExposeParam.BackupPVCConfig[csiExposeParam.StorageClass]; exists {
 		if value.StorageClass != "" {
 			backupPVCStorageClass = value.StorageClass
 		}
 
 		backupPVCReadOnly = value.ReadOnly
+		spcNoRelabeling = value.SPCNoRelabeling
```

Comment:
`spcNoRelabeling` is only required to set when `backupPVCReadOnly=true`. However, here we allow users to set it separately.
This would be a security risk strictly speaking, so would we change it to:
```
if backupPVCReadOnly == true {
     spcNoRelabeling = value.SPCNoRelabeling
}
```

Reply:
+1 sounds good, lets just document this behavior explicitly (i.e. `SPCNoRelabeling` key would be ignored if `backupPVCReadOnly` is not set to true.)